data.md

```diff
@@ -420,12 +420,12 @@ The implementation is not correct for now because the UTF-8 encoding is not impl
 
     syntax Bytes ::= #encodeUTF8 ( Int, Endianness ) [function]
  // -----------------------------------------------------------
-    rule #encodeUTF8 (I, E) => Int2Bytes(I, BE, Unsigned) requires I <=Int 127
-    rule #encodeUTF8 (I, E) => Int2Bytes(((((I &Int 1984) >>Int 6) +Int 192) <<Int 8) +Int ((I &Int 63) +Int 128), BE, Unsigned)
+    rule #encodeUTF8 (I, _E) => Int2Bytes(I, BE, Unsigned) requires I <=Int 127
```

Comment:
A bit suspicious that we don't use the endianness anwhere

Reply:
Well, I would say either remove that argument to `#encodeUTF8`, or pass it through to the second argument of `Int2Bytes`. But it shouldn't just sit there.